tnoodle-ui/src/Layout.js

```diff
@@ -95,7 +95,7 @@ function analyzeVersion(versionInfo) {
   }
   var isAllowed = versionInfo.allowed.indexOf(runningVersion) >= 0;
   var isCurrent = runningVersion === versionInfo.current.name;
-  var downloadCurrent = (<span><a href={versionInfo.current.download} target='_blank'>{versionInfo.current.name}</a> (<a href={versionInfo.current.information} target='_blank'>details</a>)</span>);
+  var downloadCurrent = (<span><a href={versionInfo.current.download} target='_blank' rel="noopener noreferrer">{versionInfo.current.name}</a> (<a href={versionInfo.current.information} target='_blank' rel="noopener noreferrer">details</a>)</span>);
```

Comment:
> Added rel noopener and noreferrer to target blank links

This commit message tells me what you did, but not *why* you did it. I can figure out what you did by looking at the diff =) I had to do some googling to figure out why we want to do this. Could you update your commit message to explain why you did what you did, not just what you did?

Also, how do you feel about creating an `ExternalLink` component that we could use across all of TNoodle? That would also give us a nice place to add a comment explaining why we want this `noopener noreferrer` stuff.